### PR TITLE
Concurrent steal tests fixed

### PR DESCRIPTION
**Pull request by Razz4780**
Following https://github.com/metalbear-co/mirrord/pull/2476

### CODE REVIEW

File: mirrord/operator/src/crd.rs [hunk 1]
```diff
@@ -21,7 +21,8 @@ pub const TARGETLESS_TARGET_NAME: &str = "targetless";
 pub struct TargetSpec {
     /// None when targetless.
     pub target: Option<Target>,
-    #[deprecated = "port locks are enforced only in the operator, new operator versions always return an empty list"]
```
Comment (meowjesty) [inline mirrord/operator/src/crd.rs:24]: Why remove this in favor of docs?

**Review by meowjesty — APPROVED** [review]
:+1: 

Have 1 question though.

File: mirrord/operator/src/crd.rs (hunk 1 above)
Comment (Razz4780) [reply mirrord/operator/src/crd.rs:24]: Too many warnings in the operator code, I got tired of copy-pasting `#[allow(deprecated)]`. But I figured I can just remove the field, it's backwards-compatible ^^

Comment (meowjesty) [reply mirrord/operator/src/crd.rs:24]: Nice!

**Pull request merged**